backend/handler/igdb_handler.py

```diff
@@ -32,75 +35,79 @@ def wrapper(*args):
 
         return wrapper
 
+    def _request(self, url: str, data: str, timeout: int = 120) -> list:
+        try:
+            res = requests.post(url, data, headers=self.headers, timeout=timeout)
+            res.raise_for_status()
+        except (HTTPError, Timeout) as err:
+            log.error(err)
+            # All requests to the IGDB API return a list
```

Comment:
This is (hopefully) the case for all the endpoints we use in this file 🤞🏼

Reply:
Yeah I think so, the IGDB API always returns a list (at least all the endpoints I tried so far)